lib/graphql_generator.dart

```diff
@@ -34,13 +35,16 @@ class GraphQLGenerators extends GeneratorForAnnotation<GQLGenerator> {
           _convertDartObjectMapToString(annotation.read('fragments').mapValue);
 
     var response = await getSchema(url, headerToken);
+    if (response != null) {
+      var typesFromResponse = getTypeList(getTypesFromResponse(response));
 
-    var typesFromResponse = getTypeList(getTypesFromResponse(response));
+      var mutationTypeName = getMutationTypeNameFromResponse(response);
 
-    var mutationTypeName = getMutationTypeNameFromResponse(response);
-
-    return GraphQLCodeGenerators().generateDartCode(
-        typesFromResponse, namespace, types, fragments, mutationTypeName);
+      return GraphQLCodeGenerators().generateDartCode(
+          typesFromResponse, namespace, types, fragments, mutationTypeName);
+    } else {
+      return null;
```

Comment:
@justinjoyn will the code generation fail silently? I think it's worth logging some error out if possible.